src/config/config.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+module.exports = {
+  production: {
+    use_env_variable: process.env.DATABASE_URL
+  },
+
+  development: {
```

Comment:
> Why not stick with **connection URI** since it was used in the configuration for the production environment

Good suggestion. Considered that too. But my choice has reasons.
Heroku uses  *connection URI*
Travis uses *key - value pair*